src/sectionproperties/analysis/fea.py

```diff
@@ -92,8 +92,8 @@ def geometric_properties(
         iyy: float = 0.0
         ixy: float = 0.0
 
-        # Gauss points for 6 point Gaussian integration
-        gps = gauss_points(n=6)
+        # Gauss points for 4 point Gaussian integration
+        gps = gauss_points(n=4)
```

Comment:
For T6, 3-point quadrature (2nd order) is sufficient. 6-point quadrature (5th order) is an overkill. Here use 4-point (3rd order) should be okay to cover most cases, even with nonlinear material. This reduces 30% time spent on elemental quantities.